skopt/optimizer/base.py

```diff
@@ -75,24 +75,24 @@ def base_minimize(func, dimensions, base_estimator,
         Function to minimize over the posterior distribution. Can be either
 
         - `"LCB"` for lower confidence bound,
-        - `"EI"` for expected improvement,
-        - `"PI"` for probability of improvement.
+        - `"EI"` for negative expected improvement,
+        - `"PI"` for negative probability of improvement.
 
     * `acq_optimizer` [string, `"auto"`, `"sampling"` or `"lbfgs"`, default=`"auto"`]:
-        Searching for the next possible candidate to update the prior
-        with.
-
-        If acq_optimizer is set to `"auto"`, then it is set to `"lbfgs"`` if
-        all the search dimensions are Real(continuous). It defaults to
-        `"sampling"` for all other cases.
-
-        If acq_optimizer is set to `"sampling"`, `n_points` are sampled randomly
-        and the Gaussian Process prior is updated with the point that gives
-        the best acquisition value over the Gaussian prior.
-
-        If acq_optimizer is set to `"lbfgs"`, then a point is sampled randomly, and
-        lbfgs is run for 5 iterations optimizing the acquisition function
-        over the Gaussian posterior.
+        Method to minimize the acquistion function. The prior over `func`
+        is updated with the optimal value obtained by optimizing `acq_func`
+        with `acq_optimizer`.
```

Comment:
For my education: we have the objective function `func` and the search space for the parameters. What is the prior?